tests/ice40/add_sub.ys

```diff
@@ -0,0 +1,7 @@
+equiv_opt -map ../../techlibs/ice40/cells_sim.v synth_ice40
+synth_ice40
+select -assert-count 12 t:SB_LUT4
+select -assert-count 7 t:SB_CARRY
+select -assert-count 2 t:$logic_and
```

Comment:
I would expect any of these internal cells here. They should go away after you do a `design -load`

Reply:
Sorry: I *wouldn't* expect...